util/coverage/coverage.py

```diff
@@ -28,6 +29,16 @@
 }
 
 
+@app.command()
+def combined_report(out_root_dir: Path = Path("coverage"),
```

Comment:
Nit: Perhaps you can verbify this function name? It would also avoid the name collision with the `combined_report` package.